dlp/snippets/deid_test.py

```diff
@@ -299,6 +299,32 @@ def test_deidentify_with_replace_infotype(capsys):
     assert "My favorite site is [URL]" in out
 
 
+def test_deidentify_with_simple_word_list(capsys):
+    deid.deidentify_with_simple_word_list(
+        GCLOUD_PROJECT,
+        "My Room is rm-green",
```

Comment:
Please use "Patient was seen in RM-YELLOW then transferred to rm green."

Reply:
updated